src/content_mapblock.cpp

```diff
@@ -1513,7 +1513,50 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 
 			v3f pos = intToFloat(p, BS);
 
-			std::vector<aabb3f> boxes = n.getNodeBoxes(nodedef);
+			u8 neighbors = 0;
+
+			// locate possible neighboring nodes to connect to
+			if (f.node_box.type == NODEBOX_CONNECTED) {
```

Comment:
seriously don't do this.

I've now spent probably 6+ hours trying to find out how to get both those ideas to work. Plus way too much developing a headache.

But I can't even get *this* to compile:

nodedef.h:
```
virtual int getConnectedNeighbors(MapNode (*getNodeNoExFn)(v3s16 p), v3s16 p)=0;
```

"because the following virtual functions are pure" ... wtf?

I've done function pointers in C, but this is just demotivating. I'm ready to scream.